### PR TITLE
KAA-1228 TCP channel ignores EOF error

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: client/client-multi/client-cpp/impl/channel/impl/DefaultOperationTcpChannel.cpp [hunk 1]
```diff
@@ -369,10 +369,12 @@ boost::system::error_code DefaultOperationTcpChannel::sendPingRequest()
...
-    boost::asio::async_read(*sock_, *responseBuffer_,
-              boost::asio::transfer_at_least(1),
-              boost::bind(&DefaultOperationTcpChannel::onReadEvent, this,
-                      boost::asio::placeholders::error));
+    boost::asio::async_read(*sock_
+                          , *responseBuffer_
+                          , boost::asio::transfer_at_least(1)
+                          , boost::bind(&DefaultOperationTcpChannel::onReadEvent
+                                      , this
+                                      , boost::asio::placeholders::error));
```
Comment (rasendubi) [inline client/client-multi/client-cpp/impl/channel/impl/DefaultOperationTcpChannel.cpp]: Please, check out [C code style guide](https://github.com/kaaproject/kaa/blob/develop/client/client-multi/client-c/docs/code-style.org).

Comment (rasendubi) [inline client/client-multi/client-cpp/impl/channel/impl/DefaultOperationTcpChannel.cpp]: Don't put commas and start of line. Sorry, it wasn't stated in the guide; I'll try to update it soon.

**Comment by forGGe** [comment]
Looks good

**Comment by forGGe** [comment]
@rasendubi , please review

**Comment by forGGe** [comment]
@GamovCoder

**Comment by rasendubi** [comment]
I don't see any issue, but I'm not familiar with the code in question.

Though, I would like to see a test showing the bug, that's not critical.

**Comment by ohamov** [comment]
LGTM

**Comment by rasendubi** [comment]
@kaa-leeroy retest this please

**Pull request merged**